.github/dependabot.yaml

```diff
@@ -0,0 +1,58 @@
+version: 2
```

Comment:
It might be nice to also add a Dependabot badge to the top of the README to make it easier to tell which tools we use at a glance.

Reply:
@mwarres discovered that the badge I found was unreliable and probably outdated, so let's omit the badge for now.